### ISSUE TITLE
Incorrect sound being played for photon emission from source

### ISSUE THREAD

**Issue by jbphet** [phetsims/molecules-and-light#396]
While working on https://github.com/phetsims/greenhouse-effect/issues/333 I discovered that the sound for emission of the photon from the molecule is being played when the photons are emitted from the photon source.  It is supposed to be a distinct sound.  It is working correctly in the published version on the website, which is version 1.5.21.

**Comment by jbphet** [comment]
@samreid - It looks to me like this problem was introduced in https://github.com/phetsims/greenhouse-effect/commit/2924ed423cdaa7276087275cbe0c3243e20c1434 and https://github.com/phetsims/greenhouse-effect/commit/02a01782f3bfad87f6727b4065fe5528674fe4c9, when the contains used for the photons was converted from an `ObservableArray` to a `PhetioGroup`.  Previously, the photon was created, initialized, then added to the observable array, and there was a listener that played different sounds based on the photon's position.  That listener was moved to the point where the photon is created by the group, which is before it is completely initialized.

As noted above, this problem does not exist in the currently published version.  I just looked at https://bayes.colorado.edu/dev/phet-io/latest/ to see if the published phet-io version had this problem, and that page links to version 1.3.22-phetio, so it's way old and doesn't even have sound.  So I guess there was some phet-io work done to update this sim, but it was never published.

@samreid - assigning to you to resolve.

**Comment by samreid** [comment]
I looked into this previously but did not see an obvious solution. This would be good to collaborate with @jbphet. We are on a subteam together, so I'll add it to our iteration priorities and track it through Monday.

**Comment by jbphet** [comment]
I addressed this by making the `createElement` function for the `photonGroup` include a parameter for the photon's initial position.  This required adding an option to `MicroPhoton` for specifying that initial position.  With this in place, the photons now have the correct positions when the `photonGroup.elementCreatedEmitter` fires, which allows `PhotonEmissionSoundGenerator` to produce the correct sound.  I tested locally and it works.  Since this problem doesn't exist in the published version there is no need for a maintenance release.  Closing.
